rootfs/api/urls.py

```diff
@@ -45,7 +45,8 @@
     url(r"^apps/(?P<id>{})/containers/?".format(settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'get': 'list'})),
     # application domains
-    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\._\w]+)/?".format(settings.APP_URL_REGEX),
+    # FIXME make sure * is only matched at the start
+    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\*\._\w]+)/?".format(settings.APP_URL_REGEX),
```

Comment:
Ah neat - I just realised I had tried it in my regex program using a Ruby type matching so that's probably telling :)

Going to add this regex in and see how it goes!